src/main/java/uk/gov/cshr/controller/signup/SignupController.java

```diff
@@ -93,11 +99,24 @@ public String sendInvite(Model model,
         }
 
         final String email = form.getEmail();
-
-        if (inviteRepository.existsByForEmailAndStatus(email, InviteStatus.PENDING)) {
-            log.info("{} has already been invited", email);
-            redirectAttributes.addFlashAttribute(ApplicationConstants.STATUS_ATTRIBUTE, email + " has already been invited");
-            return REDIRECT_SIGNUP_REQUEST;
+        Optional<Invite> pendingInvite = inviteService.findByForEmailAndStatus(email, InviteStatus.PENDING);
+        if(pendingInvite.isPresent()) {
+            if (inviteService.isInviteCodeExpired(pendingInvite.get())) {
+                log.info("{} has already been invited", email);
+                inviteService.updateInviteByCode(pendingInvite.get().getCode(), InviteStatus.EXPIRED);
+            } else {
+                long timeForReReg = new Date().getTime() - pendingInvite.get().getInvitedAt().getTime();
+                if (timeForReReg < durationAfterReRegAllowedInSeconds * 1000) {
+                    log.info("{} user trying to re-register before re-registration allowed time", email);
+                    redirectAttributes.addFlashAttribute(
+                            ApplicationConstants.STATUS_ATTRIBUTE,
+                            "You have been sent an email with a link to register your account. Please check your spam or junk mail folders.\n" +
+                                    "If you have not received your link, please wait " +
```

Comment:
replace "your link" with "the email".

Reply:
This is from the ticket. Need to check with Mo. Happy to confirm.

Reply:
DONE.
As Mo confirmed, the message is changed now.